### ISSUE TITLE
"Fixed installation" broke travis

### ISSUE THREAD

**Issue by Rayman** [tue-robotics/tue-env#18]
1514fe1fdefdc2ebfdccfe426ff9bf310f976862 introduced the following error in travis during installation: https://travis-ci.org/tue-robotics/geolib2/builds/78365757#L123
```
cat: /home/travis/.tue/user/envs/hydro: No such file or directory
...
[tue] Not all needed environment variables are set.
```
Does that error only occur on hydro?

**Comment by svddries** [comment]
Hmm, yes and Travis still uses the old way of installing. Instead it should use the bootstrap script called 'bootstrap-ros-indigo'. Ramon, can you change that?

**Comment by Rayman** [comment]
But travis runs in hydro, does that script work then?
```
Distributor ID:	Ubuntu
Description:	Ubuntu 12.04 LTS
Release:	12.04
Codename:	precise
```

**Comment by Rayman** [comment]
Fixed by 7cc99a186bb0f618c9d05536346751701abadd72 and fc6b1699e2ccd817f07e758ba15a773060c84c88
